core/arch/arm/plat-stm32mp1/drivers/stm32mp1_pmic.c

```diff
@@ -47,12 +47,14 @@ static_assert(PMIC_REGU_FLAG_COUNT <= UINT_MAX);
  * struct pmic_regulator_data - Platform specific data
  * @flags: Flags for platform property to apply
  * @regu_name: Regulator name ID in stpmic1 driver
- * @supported_voltages: Supported levels description or NULL is not yet built
+ * @voltages_desc: Supported levels description
+ * @voltages_level: Pointer to supported levels or NULL is not yet allocated
```

Comment:
`s/is not yet/if not yet allocated/`

In commit description: `s/2 argument/2 arguments/`